examples/step-32/step-32.cc

```diff
@@ -3577,9 +3577,9 @@ namespace Step32
                                 EquationData::R1,
                                 (dim==3) ? 96 : 12,
                                 true);
-    static HyperShellBoundary<dim> boundary;
-    triangulation.set_boundary (0, boundary);
-    triangulation.set_boundary (1, boundary);
+    triangulation.set_all_manifold_ids(0);
+    static SphericalManifold<dim> manifold;
+    triangulation.set_manifold (0, manifold);
```

Comment:
I'm running a full test now. The above does not give me any problems with MappingQ(4). Instead, in debug mode, it complains about the no_flux_boundary_conditions, which require a Boundary and not a manifold. I'm adding back also a boundary description on the boundary, in addition to the curved Manifold in the interior. 
I didn't see problems, because I was not running in debug mode (it does take quite a while in debug mode...).